blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueIcon.java

```diff
@@ -0,0 +1,11 @@
+package io.jenkins.blueocean.rest.model;
+
+import io.jenkins.blueocean.rest.Navigable;
+
+public abstract class BlueIcon extends Resource {
+
+    public static final int DEFAULT_ICON_SIZE = 20;
+
+    @Navigable
+    public abstract void getUrl();
```

Comment:
There is no return from this API, does redirection and thats why its void. It definitely needs documentation to explain behavior. Either way something we can address in separate PR.